build.sbt

```diff
@@ -10,7 +10,8 @@ libraryDependencies ++= Seq(
   "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.4.0" % "test",
   "junit" % "junit" % "4.13.2" % "test",
   "com.github.sbt" % "junit-interface" % "0.13.3" % "test",
-  "org.apache.logging.log4j" % "log4j-core" % "2.19.0" % "test"
+  "org.apache.logging.log4j" % "log4j-core" % "2.19.0" % "test",
+  "owl" % "xsat-owl" % "2.0.0-SNAPSHOT" % "test"
```

Comment:
If this lives on a branch (not main/master) then fine, but otherwise we shouldn't put dependencies on non-public libraries into this schema.